### DISCUSSION TITLE
Add Cosmetics Win Rate

### DISCUSSION THREAD

**Discussion by Elitesparkle** [Heroes-Profile/heroesprofile#974] [Ideas]
I'd like to see the Win Rate and Pick Rate of Mounts, Skins, Sprays, Banners, and Portraits.

I'd also like to see the Win Rate related to the amount of times Taunt, Voice Line, and Spray were used.

I don't know if it's possible to count B-Steps. If yes, I'd also like to see the Win Rate for that.

**Comment by Zemill** [comment]
We attempted to add this a few years back and the data didn't look all that interesting.  Maybe I will take a second look at it

**Comment by Zemill** [comment]
We've decided not to pursue this as the data is too scattered to make interesting deductions

**Comment by Elitesparkle** [comment]
No problem! I appreciate that you at least gave it a try locally.
